Cargo.toml

```diff
@@ -13,6 +13,7 @@ members = [
 default-members = [
     "devolutions-gateway",
     "jetsocat",
+    "crates/picky-krb",
```

Comment:
It should not be default (it's compiled as part of devolutions-gateway)

Reply:
Done